tests/pages/nodes/NodesPage-cy.js

```diff
@@ -6,7 +6,7 @@ describe("Nodes Page", function() {
         nodeHealth: true
       });
       cy.visitUrl({ url: "/nodes" });
-      cy.get("tbody tr td:first-child").as("hostnames");
+      cy.get(".BottomLeftGrid_ScrollWrapper").as("hostnames");
```

Comment:
What other selector could we use? The idea of those tests is to operate with human readable selectors. It wasn't perfect, but a programmer could say we're looking for the first cell in a table. `.BottomLeftGrid_ScrollWrapper` doesn't tell me anything.

Reply:
I could use `.table-cell-link-primary` but this is not really more verbose I would argue and it also might become an easier target to change.

The problem is that our HTML is not semantic anymore, we just use a bunch of divs and spans now plus every class coming from the ui-kit is a) randomly generated and b) changes with every release of the UI-kit

Reply:
Please create a JIRA to find a solution.